### PR TITLE
kovan-ovm-futures (refactored)

### PR DESCRIPTION
**Pull request by liamzebedee**
Refactored as the original PR had some unrelated commits in its history.

Deployed using account 0x19C6e6B49D529C2ae16dCC794f08a93dd813859D.

### CODE REVIEW

File: publish/deployed/kovan-ovm/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"SystemSettings": {
-		"deploy": false
```
Comment (zyzek) [inline publish/deployed/kovan-ovm/config.json:3]: `kovan-ovm` shouldn't be touched yet, I think.

File: publish/deployed/kovan-ovm-futures/feeds.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+	"SNX": { "asset": "SNX", "feed": "0x0" }
```
Comment (zyzek) [inline publish/deployed/kovan-ovm-futures/feeds.json:2]: We'll need feeds for `SNX`, `ETH`, `BTC`, and `LINK`. These can all be copied from the `kovan-ovm` feeds file, as we can share aggregators between the two deployments.... Or are we running an oracle manually for this deployment?

Comment (zyzek) [reply publish/deployed/kovan-ovm-futures/feeds.json:2]: Might as well add the corresponding synths to the deployment while we're at it for debt pool calculation tests.

**Review by zyzek — CHANGES_REQUESTED** [review]

**Review by zyzek — APPROVED** [review]

File: publish/deployed/kovan-ovm-futures/config.json [hunk 3]
```diff
@@ -0,0 +1,173 @@
+{
```
Comment (zyzek) [inline publish/deployed/kovan-ovm-futures/config.json:1]: If the deployment has actually completed, should these not be `false`?

File: publish/deployed/kovan-ovm-futures/synths.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[
+	{
+		"name": "sUSD",
+		"asset": "USD"
+	},
+	{
+		"name": "sETH",
```
Comment (zyzek) [inline publish/deployed/kovan-ovm-futures/synths.json:7]: Didn't we need `sBTC` and `sLINK` as well?

File: publish/deployed/kovan-ovm-futures/config.json (hunk 3 above)
Comment (zyzek) [inline publish/deployed/kovan-ovm-futures/config.json:1]: If the deployment has actually completed, should these not be `false`?

**Review by zyzek — CHANGES_REQUESTED** [review]
Is `publish/deployed/local-ovm/deployment.json ` actually supposed to be committed?

File: publish/deployed/kovan-ovm-futures/synths.json (hunk 4 above)
Comment (liamzebedee) [reply publish/deployed/kovan-ovm-futures/synths.json:7]: Do we? I thought synths.json are for synths we are enabling/deploying, whereas we just need their feeds.

File: publish/deployed/kovan-ovm-futures/config.json (hunk 3 above)
Comment (liamzebedee) [reply publish/deployed/kovan-ovm-futures/config.json:1]: Yeah I realise this now. Will update.

**Review by zyzek — APPROVED** [review]
Looks good to me.

**Pull request merged**